dask/dataframe/io/tests/test_orc.py

```diff
@@ -0,0 +1,60 @@
+import os
+import pytest
+import shutil
+import tempfile
+
+from dask.dataframe import read_orc
+from dask.dataframe.utils import assert_eq
+import dask.dataframe as dd
+from dask.compatibility import PY2
+
+pytest.importorskip('pyarrow.orc')
+url = ('https://www.googleapis.com/download/storage/v1/b/anaconda-public-data/o'
+       '/orc%2FTestOrcFile.testDate1900.orc?generation=1522611448751555&alt='
+       'media')
+columns = ['time', 'date']
+
+
+@pytest.mark.skipif(PY2, reason='moto and requests confusion')
+@pytest.mark.network
+def test_orc_with_backend():
+    d = read_orc(url)
+    assert set(d.columns) == {'time', 'date'}  # order is not guranteed
+    assert len(d) == 70000
+
+
+@pytest.fixture(scope='module')
+def orc_files():
+    requests = pytest.importorskip('requests')
+    data = requests.get(url).content
+    d = tempfile.mkdtemp()
+    files = [os.path.join(d, fn) for fn in ['test1.orc', 'test2.orc']]
+    for fn in files:
+        with open(fn, 'wb') as f:
+            f.write(data)
+    try:
+        yield files
+    finally:
+        shutil.rmtree(d, ignore_errors=True)
+
+
+@pytest.mark.skipif(PY2, reason='moto and requests confusion')
```

Comment:
Can we call `pytest.skip('moto and requests failing in Python 2')` in the orc_files fixture instead?  That way we would only have to do this once.  Does this work?  It looks like you may have tried things like this already, not sure.

Reply:
I tried that for the module as a whole, can try in the fixture instead